build/package/migrate.Dockerfile

```diff
@@ -1,16 +0,0 @@
-# Base Go environment
-# -------------------
-FROM golang:1.21-alpine as base
-WORKDIR /hatchet
-
-RUN apk update && apk add --no-cache gcc musl-dev git
```

Comment:
@abelanger5 I removed this line, please let me know if that is needed for something

Reply:
Nope, not needed 👍